### PR TITLE
Update safe-links.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to navigation and activate safelinks

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5356

File: microsoft-365/business-video/safe-links.md [hunk 1]
```diff
@@ -32,14 +32,10 @@ Microsoft Defender for Office 365 , formerly called Microsoft 365 ATP, or Advanc
...
-1. Under **Policies that apply to the entire organization**, choose the **Default** policy, and then select the **Edit** icon.
-1. Enter a URL that you want to block.
-1. Select **Use safe links in Office apps, Office for iOS and Android**; select **Do not track when users click safe links**; and select **Do not let users click through safe links to original URL**. These might already be selected if you set up the default policy. Select **Save**.
-1. Under **Policies that apply to specific recipients**, choose **Recommended safe links rule**, and then select the **Edit** icon.
-1. Select **settings**, scroll down, enter the URL that you do not want to be checked, and then select the **Add** icon.
-1. Select **applied to**, and then select your domain name. Select any additional domains that you want the rule applied to. Select **add**, **OK**, and then **Save**.
+2. Scroll down to **Increase protection from advanced threats**. Select **Manage**,and then **Safe Links**.
+3. Select **Global Settings**, in **Block the following URLs** Enter a URL that you want to block.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/safe-links.md]:
```suggestion
3. Select **Global Settings** and in **Block the following URLs**, enter the URL that you want to block.
```

File: microsoft-365/business-video/safe-links.md [hunk 2]
```diff
@@ -32,14 +32,10 @@ Microsoft Defender for Office 365 , formerly called Microsoft 365 ATP, or Advanc
...
+4. Select **Use Safe Links in Office 365 app**; select **Do not track when users click protected links in Office 365 apps**; and select **Do not let users click through to the original URL in Office 365 apps**. These might already be selected if you set up the default policy. Select **Save**.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/safe-links.md]:
```suggestion
4. Select **Use Safe Links in Office 365 app**, select **Do not track when users click protected links in Office 365 apps**, and select **Do not let users click through to the original URL in Office 365 apps**. These might already be selected if you set up the default policy. Select **Save**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**